tests/HJSONParserTest.php

```diff
@@ -56,7 +56,6 @@ private function runEach($name, $file, $isJson, $inputCr, $outputCr)
                     $this->assertEquals($json1, $json2);
                 }
             }
-            else $this->assertFalse(false);
```

Comment:
Sorry, I wasn't very clear. Please revert this line back to
```php
else $this->assertFalse(true);
```

This line is meant to catch bugs in some tests.

Reply:
@dqsully, thank you for your reply. How about using the [markTestIncomplete](https://phpunit.de/manual/6.5/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.incomplete-tests.examples.SampleTest.php) or `fail` to catch/mark the uncompleted/failed tests.

Reply:
Yeah, that's an even better solution since it can output a message. Go ahead and use that instead.

Reply:
@dqsully, thank you for your reply. I've used the `markTestIncomplete` to mark the failed test.
Please review the failed message and I think I need the suggestion to let the message be modified.

Reply:
I didn't see your comment here when I approved the PR, sorry. I updated the text to say `This test succeeded on data that should fail.`